bgp/modules/terms.py

```diff
@@ -68,24 +64,27 @@ def results(self):
 class ReadingLevelModule:
 
     def __init__(self):
-        self.isbn= None
         self.lexile_min_age= 'None'
         self.lexile_max_age= 'None'
         self.readability_fk_score= None
         self.readability_s_score= None
         self.time = 0
 
-    def run(self, doc, **kwargs):
+    def run(self, book, **kwargs):
+        doc = book.plaintext
+        isbn = 'isbn' in book.metadata and book.metadata['isbn'][0]
+
+        url = 'https://atlas-fab.lexile.com/free/books/' + str(isbn)
 
-        url = 'https://atlas-fab.lexile.com/free/books/' + str(self.isbn)
         headers = {'accept': 'application/json; version=1.0'}
         lexile = requests.get(url, headers=headers)
         # Checks if lexile exists for ISBN. If doesn't exist value remains 'None'.
         # If lexile does exist but no age range, value will be 'None'.
         # If no ISBN, value will be 'None'.
         if lexile.status_code == 200:
-            self.lexile_min_age = str(lexile.json()['data']['work']['min_age'])
-            self.lexile_max_age = str(lexile.json()['data']['work']['max_age'])
+            lexile_work = lexile.json()['data']['work']
```

Comment:
I wonder if this can fail even with a `200` response, e.g. what happens when we request the `None` url described in https://github.com/Open-Book-Genome-Project/sequencer/pull/54/files#r602396266?

e.g. https://atlas-fab.lexile.com/free/books/None

Looks like we get a 404 which is good!